package.json

```
@@ -5,8 +5,9 @@
   "main": "index.js",
   "scripts": {
     "test": "nyc --reporter=json --reporter=text --reporter=lcov mocha -R spec test/",
-    "lint": "eslint-godaddy index.js test/*.test.js",
-    "pretest": "npm run lint"
+    "lint": "eslint-godaddy *.js \"test/**/*.test.js\"",
+    "posttest": "npm run lint",
+    "localstack": "docker run -it -p 4569:4569 --rm localstack/localstack:0.10.3"
```

what about just using `localstack/localstack`, e.g.
```suggestion
    "localstack": "docker run -it -p 4569:4569 --rm localstack/localstack"
```

which should be possible with 

```
docker pull localstack/localstack:latest
```